zerver/forms.py

```diff
@@ -25,6 +27,15 @@
                     u"<a href=\"https://blogs.dropbox.com/tech/2015/09/open-sourcing-zulip-a-dropbox-hack-week-project/\">Zulip is open source</a>, so you can install your own Zulip server " + \
                     u"by following the instructions on <a href=\"https://www.zulip.org\">www.zulip.org</a>!"
 
+
+def get_registration_string(domain):
+    # type: (unicode) -> unicode
+    register_url  = reverse('register') + domain
+    register_account_string = 'The organization with the domain already exist. '+ \
```

Comment:
should say "already exists" (plural).  Also you should tag this for internationalization with gettext.